controllers/nginx/rootfs/Dockerfile

```diff
@@ -12,21 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
-
-CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+FROM gcr.io/google_containers/nginx-slim-amd64:0.23
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
   diffutils \
   --no-install-recommends \
   && rm -rf /var/lib/apt/lists/*
 
-RUN curl -sSL -o /tmp/dumb-init.deb http://ftp.us.debian.org/debian/pool/main/d/dumb-init/dumb-init_1.2.0-1_DUMB_ARCH.deb && \
-  dpkg -i /tmp/dumb-init.deb && \
-  rm /tmp/dumb-init.deb
+RUN curl -sSL -o /sbin/tini https://github.com/krallin/tini/releases/download/v0.15.0/tini-amd64 && \
+  chmod +x /sbin/tini
 
-ENTRYPOINT ["/usr/bin/dumb-init"]
+ENTRYPOINT ["/sbin/tini", "--"]
```

Comment:
I was having issues with using `dumb-init` that's why I switched to the one from shopify-docker-images. I'll see if I can rely on using the default makefile